MIAUI/Views/NewTaskPage.xaml

```diff
@@ -35,23 +36,24 @@
                         ItemsSource="{Binding Items}"
                         SelectionMode="None">
             <CollectionView.ItemTemplate>
-                <DataTemplate x:DataType="{x:Type x:String}">
+                <DataTemplate x:DataType="model:Task">
                     <SwipeView>
                         <SwipeView.RightItems>
                             <SwipeItems>
+                                ,
```

Comment:
Accident?

Reply:
Thanks for catching the extra line/comma. Removed line 43 in the new commit.